site/website/versioned_docs/version-4.1/api/serialization.md

```diff
@@ -343,7 +343,7 @@ dependency to your Jet job's project:
 <!--Gradle-->
 
 ```groovy
-compile "com.hazelcast.jet:hazelcast-jet-protobuf:${hazelcast.jet.version}"
+compile "com.hazelcast.jet:hazelcast-jet-protobuf:${jet-version}"
```

Comment:
It should be already replaced in `versioned_docs`. Please `${jet-version}` -> `4.1` for this file.